src/buza/templates/base.html

```diff
@@ -1,6 +1,7 @@
 {% load staticfiles static %}
 
 <!DOCTYPE html>
+<html>

```

Comment:
nice catch

Reply:
The one test assertion actually caught this. :)